src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -441,6 +445,16 @@ struct uct_ib_mlx5_atomic_caps_bits {
     uint8_t    reserved_at_2c0[0x540];
 };
 
+struct uct_ib_mlx5_cmd_hca_cap_2_bits {
+    uint8_t    reserved_at_0[0x80];
+
```

Comment:
- pls remove extra empty lines in structures
 - pls document fields with comments
 - maybe unite paddings like `reserved_at_N[0x80 + 0x13]`?

Reply:
@evgeny-leksikov 
- This is aligning the format as exist code in the same file, and also the similar format is used in rdma-core. The empty lines are generately used.
- Will add the comments.
- Currently just align the format as exist code in the same file, and also the similar format is used in rdma-core.

Reply:
@evgeny-leksikov 
- This is aligning the format as exist code in the same file, and also the similar format is used in rdma-core. The empty lines are generately used.
- Will add the comments.
- Currently just align the format as exist code in the same file, and also the similar format is used in rdma-core.